claymore-frontend/src/components/PaymentForm.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+
+export default function PaymentForm() {
+  return (
+    <React.Fragment>
+      <Typography variant="h6" gutterBottom>
+        Payment method
+      </Typography>
+      <Grid container spacing={3}>
+        <Grid item xs={12} md={6}>
+          <TextField
+            required
+            id="cardName"
+            label="Name on card"
+            fullWidth
+            autoComplete="cc-name"
+            variant="standard"
+          />
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <TextField
+            required
+            id="cardNumber"
+            label="Card number"
+            fullWidth
+            autoComplete="cc-number"
+            variant="standard"
+          />
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <TextField
+            required
+            id="expDate"
+            label="Expiry date"
+            fullWidth
+            autoComplete="cc-exp"
+            variant="standard"
+          />
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <TextField
+            required
+            id="cvv"
+            label="CVV"
+            helperText="Last three digits on signature strip"
+            fullWidth
+            autoComplete="cc-csc"
+            variant="standard"
+          />
+        </Grid>
+        <Grid item xs={12}>
```

Comment:
Let's not implement saving credit card details. I think it's low priority since most browsers do it for you already, and there are other things to focus on.

Reply:
We don’t need to add any more code to maintain it, and I think keeping this makes the page look better. So I’m keeping it for now.